addons/account/models/account_journal_dashboard.py

```diff
@@ -268,23 +268,10 @@ def get_journal_dashboard_datas(self):
             self.env.cr.execute(query, query_args)
             query_results_drafts = self.env.cr.dictfetchall()
 
-            today = fields.Date.context_today(self)
-            query = '''
-                SELECT
-                    (CASE WHEN move_type IN ('out_refund', 'in_refund') THEN -1 ELSE 1 END) * amount_residual AS amount_total,
```

Comment:
Wouldn't it be easier to add a line based on `amount_residual_signed` in here, and manage it correctly in `_count_results_and_sum_amounts`?

Reply:
Thanks for the input, it is in fact cleaner and easier that way!
I think there's a sign issue in v16, for a purchase journal you will see a negative value instead of positive.
I've managed this here by adding a sign paramater.
So if you agree, I guess we can forward this PR to v16

Reply:
You're right, good catch :+1: